Source/Dafny/Verifier/Translator.ClassMembers.cs

```diff
@@ -547,12 +547,12 @@ public partial class Translator {
       if (!wellformednessProc) {
         var inductionVars = ApplyInduction(m.Ins, m.Attributes);
         if (inductionVars.Count != 0) {
-          // Let the parameters be this,x,y of the method M and suppose ApplyInduction returns y.
+          // Let the parameters be this,x,y of the method M and suppose ApplyInduction returns this,y.
           // Also, let Pre be the precondition and VF be the decreases clause.
           // Then, insert into the method body what amounts to:
           //     assume case-analysis-on-parameter[[ y' ]];
-          //     forall (y' | Pre(this, x, y') && VF(this, x, y') << VF(this, x, y)) {
-          //       this.M(x, y');
+          //     forall this',y' | Pre(this', x, y') && (VF(this', x, y') << VF(this', x, y)) {
```

Comment:
```suggestion
          //     forall this',y' | Pre(this', x, y') && (VF(this', x, y') << VF(this', x, y) {
```

Reply:
That parenthesis matches the one before the first `VF`. (Since `<<` is two characters wide and isn't a Dafny operator, I felt compelled to surround the `... << ...` expression with parentheses.)

Reply:
Yes, missed that.